src/app/config/chains.ts

```diff
@@ -0,0 +1,51 @@
+import { ChainConfigs, ChainConfig } from '../../types/wallet';
+
+export const KAVA_COSMOS_CONFIG: ChainConfig = {
```

Comment:
This is very similar to our webapp chain config object, but there are a few notable differences. Most of the webapp custom items are currently not included (better to pipe them over as needed). This config aligns with `wallet_addEthereumChain` call pretty closely which is why items like `rpcUrls` are arrays.